docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -84,118 +90,170 @@ Golden Signals are a form of telemetry that applies to anything with throughput.
->   <li>Signals that are collected, but not exposed in any prebaked dashboard nor used by any alert, are candidates for removal.</li>
-> </ul>
-> [https://sre.google/sre-book/monitoring-distributed-systems/](https://sre.google/sre-book/monitoring-distributed-systems/)
+  1.  #### Each service should have enough monitors to accurately describe when it is operating **correctly** and when it is not, no more. **Ideally** **1-3 SLIs per user journey** [[source](https://www.datadoghq.com/videos/solving-reliability-fears-with-service-level-objectives/#focus-on-one-to-three-slis)].
+      <details>
+        <summary>**As Simple as Possible, No Simpler**</summary>
 
+        [From Google's SRE Book](https://sre.google/sre-book/monitoring-distributed-systems/):
+        <blockquote>
+          To avoid an overly complex, and eventually unmaintainable monitoring system we propose avoiding unnecessary complexity, such as:<br/>
+          <ul>
+            <li>Alerts on different latency thresholds, at different percentiles, on all kinds of different metrics</li>
+            <li>Extra code to detect and expose possible causes</li>
+            <li>Associated dashboards for each of these possible causes</li>
+          </ul>
+          The sources of potential complexity are never-ending. Like all software systems, monitoring can become so complex that it’s fragile, complicated to change, and a maintenance burden.
+          Therefore, design your monitoring system with an eye toward simplicity. In choosing what to monitor, keep the following guidelines in mind:
+          <ul>
+            <li>The rules that catch real incidents most often should be as simple, predictable, and reliable as possible.</li>
+            <li>Data collection, aggregation, and alerting configuration that is rarely exercised (e.g., less than once a quarter for some SRE teams) should be up for removal. </li>
+            <li>Signals that are collected, but not exposed in any prebaked dashboard nor used by any alert, are candidates for removal.</li>
+          </ul>
+        </blockquote>
+      </details>
+  </Steps>
 
-## Datadog - Monitoring Platform
+## Theory into Practice
 
 ### SLIs
 
-Generally, the SLIs should be the _Golden Signals_ for a given service, with one exception: SLIs can be a composite of other SLOs, and this is how an overall site reliability score is achieved. E.g by aggregating and weighting the SLOs into an SLI.
+We should create monitors for the [_Golden Signals_](#golden-signals-slis) for a given service. These are the SLIs.
 
-SLIs are always a percentage out of 100%. Percentages are easy to understand by the widest audience and they are relative.
+We can make SLIs a percentage out of 100%.
+<img src="/assets/refarch/image-20211006-041301.png" width={"40%"} style={{"display": "block", "marginLeft": "auto", "marginRight": "auto" }} /><br/>
```

Comment:
I wanted the image centered and not full size. Thoughts on how we want to standardize this?